include/utility/json_utility.hpp

```diff
@@ -606,5 +606,25 @@ inline std::string
     }
     return std::string();
 }
+
+/**
+ * @brief An API to check backup and restore VPD is required.
+ *
+ * @param[in] i_backupAndRestoreCfgFilePath - Backup & restore config JSON file
+ * path.
+ *
+ * @return true if backup and restore config JSON file exists, false otherwise.
+ */
+inline bool
+    isBackupAndRestoreRequired(const std::string& i_backupAndRestoreCfgFilePath)
```

Comment:
If you just pass the string here and just validate the path then it qualifies for general utility not JSON utility as there is nothing w.r.t JSON in this API. 
Whereas, If you pass the JSON object here and extract the info from JSON and then do the checking then it qualifies for JSON utility.